netutils/iperf/iperf.c

```diff
@@ -267,9 +267,10 @@ static void iperf_report_task(FAR void *arg)
       printf("%7.2lf-%7.2lf sec %10ju Bytes %7.2f Mbits/sec\n",
              ts_diff(&last, &start),
              ts_diff(&now, &start),
-             now_len,
-             (((double)(now_len - last_len) * 8) /
-             ts_diff(&now, &last) / 1e6));
+             now_len -last_len,
+             (double)((now_len - last_len) * 8 / 1000000) /
```

Comment:
should we cast to double before divide 1000000?

Reply:
The result would be like this:

```
 mode=tcp-server sip=10.0.2.15:5001,dip=0.0.0.0:5001, interval=1, time=0
accept: 10.0.2.2,57866

           Interval         Transfer         Bandwidth

   0.00-   1.01 sec   38229664 Bytes     inf Mbits/sec
   1.01-   2.02 sec   40237600 Bytes  318.71 Mbits/sec
   2.02-   3.03 sec   38111840 Bytes     inf Mbits/sec
   3.03-   4.04 sec   37212480 Bytes     inf Mbits/sec
   4.04-   5.05 sec   37773120 Bytes  299.19 Mbits/sec
```

Reply:
Hi, @masayuki2009, have you ever triggered this problem (`inf`) these days? I found it still re-producing on my side with newest master, and tried to fix it by https://github.com/apache/nuttx/pull/9794.